src/renderer/i18n/ru/wallet.ts

```diff
@@ -40,7 +40,10 @@ const wallet: WalletMessages = {
   'wallet.errors.address.invalid': 'Address is invalid - RU',
   'wallet.errors.amount.shouldBeNumber': 'Количество должно быть числом',
   'wallet.errors.amount.shouldBeGreaterThan': 'Количество должно быть больше, чем {amount}',
-  'wallet.errors.amount.shouldBeLessThanBalance': 'Количество должно быть меньше вашего баланса'
+  'wallet.errors.amount.shouldBeLessThanBalance': 'Количество должно быть меньше вашего баланса',
+  'wallet.errors.amount.shouldBeLessThanBalanceAndFee':
+    'Количество должно быть меньше, чем ваш баланс после вычетом комисси',
```

Comment:
sorry. my bad. missed a typo in a translation
```
    'Количество должно быть меньше, чем ваш баланс после вычета комисси',
```

Reply:
np - fixed!